doc/release-process.md

```diff
@@ -183,9 +183,14 @@ Non-codesigners: wait for Windows/macOS detached signatures:
 - Once the Windows/macOS builds each have 3 matching signatures, they will be signed with their respective release keys.
 - Detached signatures will then be committed to the [bitcoin-detached-sigs](https://github.com/bitcoin-core/bitcoin-detached-sigs) repository, which can be combined with the unsigned apps to create signed binaries.
 
-Create (and optionally verify) the codesigned outputs:
+Create the codesigned build outputs:
 
-- [Codesigning](/contrib/guix/README.md#codesigning)
+- [Codesigning build outputs](/contrib/guix/README.md#codesigning-build-outputs)
+
+Verify other builders' signatures to your own (Optional):
```

Comment:
Is there any particular reason to capitalize "Optional"?

Reply:
fixed